lib/ansible/modules/yum.py

```diff
@@ -1322,7 +1322,7 @@ def latest(self, items, repoq):
         updates = {}
         obsoletes = {}
         update_all = False
-        cmd = None
+        cmd = self.yum_basecmd
```

Comment:
Since this is the last method called by `ensure`, the side effect doesn't matter, but cmd could be a copy of the list so self.yum_basecmd isn't mutated.

Reply:
Ah, good point. I'll fix that 👍